runtime/runtime/src/lib.rs

```diff
@@ -2450,7 +2450,9 @@ mod tests {
         assert_eq!(final_account_state.storage_usage(), 0);
     }
 
+    // This test fails on aarch because compilation results are not cached.
     #[test]
+    #[cfg_attr(all(target_arch = "aarch64", target_vendor = "apple"), ignore)]
```

Comment:
You don’t need to test for target_vendor here. aarch64 systems can't run wasmer2 either and therefore their cache will be empty too.

In fact this should be checking for [`not(target_arch = "x86_64")`](https://github.com/near/nearcore/blob/0d8f45b7c3fea3b72bb152a43842cd7625650781/runtime/near-vm-runner/src/vm_kind.rs#L44).

Reply:
Ok, I switched to checking for "x86_64" instead.